glue-exp/importers/vizier/vizier.py

```diff
@@ -0,0 +1,77 @@
+# This plugin requires OpenCV to be installed, and returns a PIL Image object
+
+import os
+import sys
+from io import BytesIO
+
+import requests
+from astropy.io.votable import parse, parse_single_table
+
+from PyQt4.QtGui import QDialog, QApplication, QTreeWidgetItem, QHeaderView
+from PyQt4.uic import loadUi
+from PyQt4.QtCore import Qt
+
+UI_MAIN = os.path.join(os.path.dirname(__file__), 'vizier.ui')
+
+
+class VizierImporter(QDialog):
+
+    def __init__(self):
+
+        super(VizierImporter, self).__init__()
+
+        self.ui = loadUi(UI_MAIN, self)
+
+        self.cancel.clicked.connect(self.reject)
```

Comment:
@ChrisBeaumont - in the case of the OK button for example, are you suggesting that I use ``connect_bool_button`` instead of doing:

        self.ok.clicked.connect(self.finalize)

?